test/specs/chart/SankeySpec.js

```diff
@@ -148,16 +148,18 @@ describe('<Sankey />', () => {
       ...data,
       nodes: [
         ...data.nodes,
-        { name: "New Node" }
+        { name: 'New Node' }
       ],
       links: [
         ...data.links,
         { source: 2, target: data.nodes.length, value: 100.0 }
       ]
     };
     wrapper.setProps({ data: newData });
+    setTimeout(() => {
```

Comment:
ComponentWillReceiveProps is synchronously, that's why the setTimeOut wasn't needed.
ComponentDidUpdate is asynchronously, probably less time would be enough.